specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,102 @@
+| Security                  | Credentials                       | Information used to authenticate users and systems, including secrets, certificates, tokens, and other keys.                                                                                           |
+| Security                  | SIEM                              | Security Information and Event Management (SIEM) systems collect and analyze security data from various sources to automatically detect and respond to threats.                                        |
+| Security                  | Threat Detection                  | Identification of potential security threats and vulnerabilities within a cloud environment using various techniques, including anomaly detection, signature-based detection, and behavioral analysis. |
+| Security                  | Other (Security)                  | Security services that do not fall into one of the defined subcategories.                                                                                                                              |
+| Storage                   | Backup                            | Secondary storage to protect against data loss.                                                                                                                                                        |
+| Storage                   | Block                             | High performance, low latency storage that provides random access.                                                                                                                                     |
+| Storage                   | File                              | Scalable, sharable storage for file-based data.                                                                                                                                                        |
+| Storage                   | Object                            | Highly available, durable storage for unstructured data.                                                                                                                                               |
+| Storage                   | Storage Transfer                  | Secure, high-speed, and offline transfer of stored data from one location to another.                                                                                                                  |
+| Storage                   | Other (Storage)                   | Storage services that do not fall into one of the defined subcategories.                                                                                                                               |
+| Web                       | Other (Web)                       | All Web services.                                                                                                                                                                                      |
```

Comment:
We need to break this down more. There's a lot in this space. More than storage.
- `Application Infrastructure`
- `Application Services`
- `Domain Management`
- `Frameworks and Templates`

Reply:
Same comments as above. Can come post 1.1 after spending lots of time driving consensus (hopefully based on some signals from customers as well)

Reply:
Aligned on path forward per discussions and meetings over the past week, and a path forward has been committed separately.  Resolving.